### DISCUSSION TITLE
I need help understanding your scripts in memory game project, scince i cant manage to deploy it on render.

### DISCUSSION THREAD

**Discussion by yyyasmin** [community/community#53995] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I need help understanding your scripts in memory game project, scince i cant manage to deploy it on render. Is there a way you could explain it / show how to deploy the project to render ?

**Comment by airtower-luna** [comment, marked as answer]
It looks like you're looking for help with a software hosted on GitHub. In that case you should look for support information *in the repository for that software*, usually in the README or a file called SUPPORT. If those don't say anything, opening an issue or discussion there is usually good. :slightly_smiling_face:
